app/presenters/check_answers/disbursement_costs_card.rb

```diff
@@ -0,0 +1,18 @@
+module CheckAnswers
+  class DisbursementCostsCard < Base
+    attr_reader :disbursement_type_form
+
+    KEY = 'disbursement_costs'.freeze
+    GROUP = 'about_claim'.freeze
+
+    def initialize(_claim)
+      # @disbursement_type_form = Steps::DisbursementTypeForm.build(claim)
```

Comment:
Do we need this? Or can the comment be removed

Reply:
removed